src/Extensions/Compatability/Logger/ULSLogging.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Concurrent;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Omex.Extensions.Compatability.Logger
+{
+	/// <summary>
+	/// Unified logging, allows registering for logging events sent by
+	/// code implementations and to raise logging events
+	/// </summary>
+	public static class ULSLogging
```

Comment:
Do we want to use the term ULS externally, given that this is essentially internal jargon whose name is now mostly meaningless? Can we just call this `UnifiedLogging`?

Reply:
All of the Compatability package provided to simplify move from our old system to new packages, that's why I use old names. The plan is that you will start using a generic host with our extensions and Compatability package to be able to build old code (namespaces still need to be changed) but use the new infrastructure, eventually you will update all code to use DI intead of static classes and then stop using Compatibility package.